docs/en/observability/apm.asciidoc

```diff
@@ -0,0 +1,12 @@
+[[apm]]
+= Application performance monitoring (APM)
+
+Elastic APM is an application performance monitoring system built on the Elastic Stack.
+It allows you to monitor software services and applications in real-time, by
```

Comment:
```suggestion
It allows you to monitor software services and applications in real time, by
```

Reply:
minor, but typically it's real time (noun) and real-time (adj).

Reply:
BTW, I opened an issue in the style repo because this is a common mistake.

Reply:
TIL